### DISCUSSION TITLE
venus os 3.01

### DISCUSSION THREAD

**Discussion by steve0564** [sfstar/hass-victron#120] [Q&A]
Actually the newest Venus-OS-version from victron is the 3.01.
Will this version work with HASS-victron?

Because of the documentation (Hass-victron) says, it won't work! (all >3.xx)

**Comment by H3xF2x** [comment]
Here v0.1.5 works with a Cerbo GX v3.01. 
Typical setup with MP2, Lynx, MPPT, Shunt.
